### PR TITLE
set summarizer write to tempfile

### PR DESCRIPTION
**Pull request by MichaelClifford**
Follow up to #302 

Noticed summarizer was having write issues in container. Using the `/tmp/` directory should address it.

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
Similar issue encountered in: #307 but this solves it a better way

**Review by Gregory-Pereira — APPROVED** [review]
/lgtm

**Comment by rhatdan** [comment]
There is security risks in writing to tmp, can you use a tmpfile name rather then a hard coded file name?

**Comment by MichaelClifford** [comment]
Thanks @rhatdan Changed it to use a tempfile instead.

File: recipes/natural_language_processing/summarizer/app/summarizer.py [hunk 1]
```diff
@@ -49,12 +50,12 @@ def read_file(file):
     file_type = file.type
     
     if file_type == "application/pdf":
-        with open(file.name, "wb") as f:
+        temp = tempfile.NamedTemporaryFile()
```
Comment (rhatdan) [inline recipes/natural_language_processing/summarizer/app/summarizer.py:53]: You still need to remove this when you are done with it?

Comment (MichaelClifford) [reply recipes/natural_language_processing/summarizer/app/summarizer.py:53]: When I tested it the tempfile deleted after the file was no longer being used.

Comment (rhatdan) [reply recipes/natural_language_processing/summarizer/app/summarizer.py:53]: Is this some magic in Python?

Comment (rhatdan) [reply recipes/natural_language_processing/summarizer/app/summarizer.py:53]:
```
Use the [tempfile module](http://docs.python.org/2/library/tempfile.html); it creates temporary files that auto-delete.

From the [tempfile.NamedTemporaryFile() documentation](http://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile):

    If delete is true (the default), the file is deleted as soon as it is closed.
```
Cool

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**